server/graphql/loaders/comment-reaction.ts

```diff
@@ -0,0 +1,17 @@
+import DataLoader from 'dataloader';
+import models from '../../models';
+import { Op } from 'sequelize';
+
+/**
+ * Loader for all collective's payout methods
+ */
+export default {
+  generateCommentReactionsLoader: (): DataLoader<number, object> =>
+    new DataLoader(async (commentIds: number[]) => {
+      const commentReactions = await models.CommentReaction.findAll({
+        where: { CommentId: { [Op.in]: commentIds } },
+      });
+
+      return commentReactions;
```

Comment:
We need to group the results by comment ID for dataloader to do its job here, check the `sortResults` helper